tomography_preprocessing/utils/aretomo.py

```diff
@@ -0,0 +1,185 @@
+import os
+from functools import lru_cache
+from pathlib import Path
+from typing import Tuple, Callable, Dict, Any
+
+import numpy as np
+import pandas as pd
+import starfile
+
+from .. import utils
+from ..utils.transformations import S, Rx, Ry, Rz
+
+def read_tlt(file: os.PathLike) -> np.ndarray:
+    """Read an IMOD tlt file into an (n, ) numpy array."""
+    return np.loadtxt(fname=file, dtype=float).reshape(-1)
+
+@lru_cache(maxsize=100)
+
+def get_xf_file(imod_directory: Path, tilt_series_id: str) -> Path:
+    """Get the xf file containing image transforms from an IMOD directory."""
+    return imod_directory / f'{tilt_series_id}.xf'
+
+
+def get_tlt_file(imod_directory: Path, tilt_series_id: str) -> Path:
+    """Get the tlt file containing tilt angles from an IMOD directory."""
+    return imod_directory / f'{tilt_series_id}.tlt'
+
+
+def get_tilt_series_alignment_parameters(
+        imod_directory: Path,
+        tilt_series_id: str
+) -> Tuple[np.ndarray, np.ndarray]:
+    """Get the tilt-series alignment parameters from an IMOD directory.
+
+    Shifts are in pixels and should be applied before rotations.
+    Rotations are ZYZ intrinsic Euler angles which transform the volume
+    """
+    tilt_angles = read_tlt(get_tlt_file(imod_directory,tilt_series_id))  
+    xf = utils.imod.read_xf(get_xf_file(imod_directory,tilt_series_id))
+    shifts_px = utils.imod.get_xf_shifts(xf)
```

Comment:
these things appear to be repeated from the IMOD utils - I would prefer to keep a single source of truth for these things and import them in here for any aretomo specific code which needs them :)

Reply:
will clean up and commit as 'aretomo util clean up'!

Reply:
removed get_* functions and read_tlt, have to duplicate get_tilt_series_alignment function as there is no etomo basename for get_xf_file and get_tlt_file etc. to work correctly.